chunky/src/java/se/llbit/chunky/ui/render/PostprocessingTab.java

```diff
@@ -97,4 +107,20 @@ public PostProcessingFilter fromString(String string) {
       controller.getCanvas().forceRepaint();
     });
   }
+
+  /**
+   * Fake post processing filter that is also a seperator for the combobox.
+   */
+  private static class PostprocessingSeparator extends Separator implements PostProcessingFilter {
```

Comment:
Note: This is only required to make it type-safe. We could just ignore the generic type of `getItems()` but that would require us to ignore warnings.